sublimate/sublimate.py

```diff
@@ -0,0 +1,176 @@
+import networkx as nx
+
+
+class victimNode:
+
+    def __init__(self, ip):
+
+        # Init the variables
+        self.ip = ip
+
+        # a list of paths, sorted by weight
+        self.compromisePaths = []
+
+    def addPath(self, path):
+
+        # If first path just append
+        if(len(self.compromisePaths) == 0):
+            self.compromisePaths.append(path)
+
+        else:
+
+            # Loop through the existing paths
+            for i in range(0,len(self.compromisePaths)):
+
+                # See if the weight of the new path is higher
+                # If yes, insert into list
+                if(path.weight > self.compromisePaths[i].weight):
+                    self.compromisePaths.insert(i, path)
+
+                # If it is the new worst path, insert at end
+                elif(i == len(self.compromisePaths)):
+                    self.compromisePaths.append(path)
```

Comment:
This could insert _path_ zero or multiple times. Python has a for-else syntax that would work:
```suggestion
                if(path.weight > self.compromisePaths[i].weight):
                    self.compromisePaths.insert(i, path)
                    break

            # If it is the new worst path, insert at end
            else(i == len(self.compromisePaths)):
                self.compromisePaths.append(path)
```

Reply:
This change looks good, didn't know about for-else.